source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
+  font-weight: bold;
```

Comment:
Yeah, not sure how I didn't pick up on that as a thing when we added it to service manual. Would be good to fix it there too, given it's probably a one liner. Letter spacing feels like a good solution… assume adding 0.03em to the normal weight rather than _condensing_ the bold version by 0.03em is more legible?